src/bin/nativelink.rs

```diff
@@ -45,7 +55,7 @@ use nativelink_service::cas_server::CasServer;
 use nativelink_service::execution_server::ExecutionServer;
 use nativelink_service::health_server::HealthServer;
 use nativelink_service::worker_api_server::WorkerApiServer;
-use nativelink_store::default_store_factory::store_factory;
```

Comment:
@allada and @aaronmondal you two are going to need to sign off on moving away from `store_factory`. I would say this new name is more self-describing for sure.

Reply:
The big thing here guys is that `make_and_add_store_to_manager ` enforces that all factory-built stores are part of a StoreManager so it's less likely that an unmanaged Store would be hanging around. Still possible to build them separately and pass into `add_store` this way though (useful for testing and probably other reasons). The old `store_factory` is still there and used by the new `make_and_add_store_to_manager ` function, but `store_factory` is no longer public in this changeset.